rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -24,13 +24,15 @@
 import co.rsk.peg.AddressBasedAuthorizer;
 import co.rsk.peg.Federation;
 import java.util.List;
+
+import co.rsk.peg.StandardMultisigFederation;
 import org.ethereum.config.blockchain.upgrades.ActivationConfig;
 import org.ethereum.config.blockchain.upgrades.ConsensusRule;
 
 public abstract class BridgeConstants {
     protected String btcParamsString;
 
-    protected Federation genesisFederation;
+    protected StandardMultisigFederation genesisFederation;
```

Comment:
Unless there's a good reason, I would always try to keep the declaration as the base object. `Federation` in this case